source/marc/enums.ttl

```diff
@@ -1603,13 +1603,13 @@
 #     skos:prefLabel "Accompanying Matter (OBSOLETE)"@en,
 #         "Accompanying Matter (OBSOLETE)"@en .
 
-:VisualRunningTimeType a :CollectionClass ;
-    :category :pending ;
-    rdfs:subClassOf :EnumeratedTerm ;
-    skos:broadMatch kbv:Visual ;
-    skos:notation "VisualRunningTimeType" ;
-    skos:prefLabel "Running Time"@en,
-        "Speltid"@sv .
+# :VisualRunningTimeType a :CollectionClass ;
+#     :category :pending ;
+#     rdfs:subClassOf :EnumeratedTerm ;
+#     skos:broadMatch kbv:Visual ;
+#     skos:notation "VisualRunningTimeType" ;
+#     skos:prefLabel "Running Time"@en,
+#         "Speltid"@sv .
```

Comment:
Sure we don't want to just erase it?

Reply:
Well we have taken the cautious path with these, since this one never had been used it might be less of a worry.